### PR TITLE
IDVA5-494-adding-health-checks

### PR DESCRIPTION
**Pull request by abhiahirv1**
Adding health-check

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by sardhani-ch — DISMISSED** [review]

File: src/main/controllers/features/healthCheckController.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { Request, Response } from "express";
```
Comment (swalke-ch) [inline src/main/controllers/features/healthCheckController.ts:1]: location of file feature/HealthCheckController.ts @sampankumar Do you think this should here? I am bit confused as well considering structure of the project.
I think if feature/ should be followed by feature name

Comment (sampankumar) [reply src/main/controllers/features/healthCheckController.ts:1]: Agreed it can come out of feature folder as well and stays in controller folder directly

Comment (sampankumar) [reply src/main/controllers/features/healthCheckController.ts:1]: Also I just noticed that the unit tests are missing

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by swalke-ch — APPROVED** [review]

**Review by sampankumar — APPROVED** [review]

**Pull request merged**